quickcache/quickcache_helper.py

```diff
@@ -131,7 +138,7 @@ def get_cache_key(self, *args, **kwargs):
     def skip(self, *args, **kwargs):
         if not self.skip_arg:
             return False
-        elif isinstance(self.skip_arg, basestring):
+        elif isinstance(self.skip_arg, six.string_types):
```

Comment:
why `six.string_types` here but `six.text_type` above? In the original, `basestring` is used in both places.

Reply:
This line is handling both text and bytes; the conversion was done automatically by `libmodernize`.

Above handles text (unicode) in the first branch (line 88), and bytes in the second branch (line 90), so they are handled separately (that code was written by yours truly, it was not an automatic conversion).

Reply:
Oh, I see now. Looks good